modules/node_modules/@ncigdc/utils/generated-relay-query-parts/fragmentGenerator.js

```diff
@@ -0,0 +1,71 @@
+/* globals _, fetch */
+/* eslint prefer-const:0, no-console:0 */
+
+let getFacetType = facet => {
+  if (_.includes(facet.field, 'datetime')) {
+    return 'datetime';
+  } else if (_.some(['_id', '_uuid', 'md5sum', 'file_name'], idSuffix => _.includes(facet.field, idSuffix))) {
+    return 'id';
+  } else if (facet.type === 'long') {
+    return 'range';
+  } else {
+    return 'terms';
+  }
+};
+
+let escapeForRelay = (string) => string.replace(/\./g, '__');
```

Comment:
import this https://github.com/NCI-GDC/portal-ui/pull/1251/files#diff-038c6caa6fcc331de947989a9c61e1d0R2?